webapp/app/translations/de/LC_MESSAGES/messages.po

```diff
@@ -2809,7 +2792,100 @@ msgstr "Geben Sie an seit wann Sie als Paar dauernd getrennt leben."
+
+#: app/forms/validations/validators.py:53
+msgid "validate.integer-length-max"
+msgstr "Das Feld darf maximal %(max)s Ziffern lang sein."
+
+#: app/forms/validations/validators.py:55
+msgid "validate.integer-length-min"
+msgstr "Das Feld muss mindestens %(min)s Ziffern lang sein."
+
+#: app/forms/validations/validators.py:57
+msgid "validate.integer-length-between"
+msgstr "Das Feld muss zwischen %(min)s und %(max)s Ziffern lang sein."
+
+#: app/forms/validations/validators.py:75
+msgid "validate.invalid-iban"
+msgstr "Ungültige IBAN Nummer"
+
+#: app/forms/validations/validators.py:78
+msgid "validate.country-code-not-de"
+msgstr "Geben Sie bitte eine IBAN an, die mit DE beginnt."
+
+#: app/forms/validations/validators.py:93
+#: app/forms/validations/validators.py:103
+#: app/forms/validations/validators.py:106
+#: app/forms/validations/validators.py:109
+#: app/forms/validations/validators.py:112
+msgid "validate.invalid-idnr"
+msgstr "Keine korrekte Steuer-Identifikationsnummer. Prüfen Sie Ihre Angabe."
+
+#: app/forms/validations/validators.py:96
+msgid "validate.idnr-length"
+msgstr ""
+"Unvollständige Angabe. Ihre Steuer-Identifikationsnummer muss aus "
+"insgesamt 11 Ziffern bestehen. Die Angabe muss folgendem Muster "
+"entsprechen: 00 000 000 000"
+
+#: app/forms/validations/validators.py:118
+#: app/forms/validations/validators.py:123
+msgid "validate.unlock-code-length"
+msgstr ""
+"Ihr Freischaltcode ist nicht vollständig. Der Freischaltcode muss aus 3 x"
+" 4 Zeichen bestehen."
+
+#: app/forms/validations/validators.py:126
+msgid "validate.invalid-unlock-code"
+msgstr "Der Freischaltcode ist nicht valide."
+
+#: app/forms/validations/validators.py:136
+#: app/forms/validations/validators.py:146
+msgid "validate.invalid-character"
+msgstr "Das Feld enthält ein Zeichen, das nicht unterstützt wird."
+
+#: app/forms/validations/validators.py:158
+msgid "validate.invalid-tax-number-length.shorter-than-ten"
+msgstr "Die Angabe muss mindestens 10 Ziffern umfassen."
+
+#: app/forms/validations/validators.py:160
+msgid "validate.invalid-tax-number-length.shorter-than-eleven"
+msgstr "Die Angabe muss mindestens 11 Ziffern umfassen."
+
+#: app/forms/validations/validators.py:162
+msgid "validate.invalid-tax-number-length.too-long"
+msgstr "Die Angabe darf höchstens 11 Ziffern umfassen."
+
+#: app/forms/validations/validators.py:171
+msgid "validate.invalid-hessen-tax-number"
+msgstr ""
+"In Hessen gibt es verschiedene Formate für die Steuernummer. Sollte die "
+"Ihnen vorliegende Nummer 10 Ziffern haben, tragen Sie bitte an erster "
+"Stelle eine 0 ein."
+
+#: app/forms/validations/validators.py:179
+msgid "validate.invalid-tax-number"
+msgstr "Die Steuernummer ist nicht valide."
+
+#: app/forms/validations/validators.py:196
+msgid "validate.disability_degree.invalid_number"
+msgstr ""
+"Keine gültige Angabe. Mögliche Werte: 20, 25, 30, 35, 40, 45, 50, 55, 60,"
+" 65, 70, 75, 80, 85, 90, 95, 100."
```

Comment:
Yes, I did not know about this comment. Thanks a lot

Reply:
Could you have another look at that?

Reply:
I think there's still something missing/has the wrong text: `Sie haben einen Anspruch auf den Pauschbetrag erst ab einem Grad der Behinderung von 20 ...` should be: `Es muss ein Grad der Behinderung von mindestens 20 angegeben werden, wenn Sie das Merkzeichen G auswählen.` or `Es muss ein Grad der Behinderung von mindestens 20 angegeben werden, wenn Sie das Merkzeichen aG auswählen. `

Reply:
Thanks, that is exactly the one where I was not sure what to do because there are so many different texts for that. However, I think I get it now. I did the following:
- If nothing selected: show old error message for GdB<20
- If G or aG selected: show the error messages you just posted.